### ISSUE TITLE
Possible error in line 418

### ISSUE THREAD

**Issue by DeWitt3942** [kriswiner/MPU9250#145]
In the line 418, in the comment, it's said it should clear bits [1:0], but  c = c & ~0x02; resets just second bit, it should be c = c & ~0x03, shouldn't it?

**Comment by kriswiner** [comment]
Yes, it looks like this should be  c = c & ~0x03, but I doubt it will make a difference in device behavior.
